azure_media_services/ams.py

```diff
@@ -417,4 +415,5 @@ def embed_player(request, usage_key_string):
     if block.share != 'off':
         return render_xblock(request, usage_key_string, check_if_enrolled=False)
 
```

Comment:
can you check if there is any cookies being written with this anonymous page and if there is any they are all essential cookies?